core/services/workflows/syncer/fetcher.go

```diff
@@ -100,5 +100,9 @@ func (s *FetcherService) Fetch(ctx context.Context, url string) ([]byte, error)
 		return nil, err
 	}
 
+	if payload.ExecutionError {
+		return nil, fmt.Errorf("execution error from gateway: %v", payload.ErrorMessage)
```

Comment:
Nit: use %s rather than %v here

Reply:
😅 forgot to change that one, done! thanks